django_usaid_clinicfinder/clinicfinder/tasks.py

```diff
@@ -223,3 +225,59 @@ def run(self, lookuppointofinterest_id, **kwargs):
                 exc_info=True)
 
 location_finder = Location_Finder()
+
+
+class PointOfInterest_Importer(Task):
+
+    """
+    Task to take dict import the data
+    """
+    name = "clinicfinder.tasks.pointofinterest_importer"
+
+    class FailedEventRequest(Exception):
+
+        """
+        The attempted task failed because of a non-200 HTTP return
+        code.
+        """
+
+    def run(self, poidata, **kwargs):
+        """
+        Returns count of imported records
+        """
+        l = self.get_logger(**kwargs)
+
+        l.info("Processing new point of interest import data")
+        imported = 0
+        try:
+            for line in poidata:
+                if "Latitude" in line and "Longitude" in line:
+                    poi_point = Point(float(line["Longitude"]), float(line["Latitude"]))
+                    # check if point exists
+                    locations = Location.objects.filter(point=poi_point)
```

Comment:
Is there anything that prevents the same location from being added multiple times with slightly different coordinates? Perhaps for future improvement check if there's another clinic within 500m with the same name?